public/assets/js/allcount.js

```diff
@@ -154,7 +158,10 @@ allcountModule.config(["fieldRenderingServiceProvider", function (fieldRendering
                     locale: $locale.id.split("-")[0],
                     format: toMomentDateFormat(dateFormat(fieldDescription))
                 });
-                input.data("DateTimePicker").date(parseDate(controller.$viewValue) || null);
+                if(typeof controller.$viewValue != 'undefined') {
```

Comment:
@stuhol Why not just `controller.$viewValue`? Are `null` or `0` appropriate values here? Also `parseDate` already has protection against value absence: https://github.com/allcount/allcountjs/pull/140/files#diff-5be2946f54f08d277a71a12fdc822ba4R54